README.md

```diff
@@ -1,16 +1,16 @@
-# Exchange Union
+# XUD
 
 [![Gitter chat](https://img.shields.io/badge/chat-on%20gitter-rose.svg)](https://gitter.im/exchangeunion/Lobby)
 [![Build Status](https://travis-ci.org/ExchangeUnion/xud.svg?branch=master)](https://travis-ci.org/ExchangeUnion/xud)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/87238907485845eb879bd61c65561775)](https://www.codacy.com/app/sangaman/xud?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ExchangeUnion/xud&amp;utm_campaign=Badge_Grade)
 
-[Exchange Union](https://www.exchangeunion.com/) (XU) is a decentralized exchange layer built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable trustless and instant cryptocurrency swaps and order fulfillment between exchanges. This repository contains the Exchange Union Daemon (`xud`) software which encompasses the following components:
+XUD stands for Exchange Union Daemon and powers [Exchange Union](https://www.exchangeunion.com/), a decentralized exchange built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable instant & trustless cryptocurrency swaps and order fulfillment between exchanges. This repository contains `xud` which encompasses the following components:
```

Comment:
If we go with "Exchange Union Daemon" for the title, this line can begin "The Exchange Union Daemon (`xud`)" or something along those lines. Also it sounds a bit awkward to say "a decentralized *exchange* built on the Lightning and Raiden network to enable instant & trustless cryptocurrency swaps and order fulfillment between *exchanges*" where "exchange" appears twice in the same phrase. Maybe something like "... enable instant and trustless cryptocurrency swaps and order fulfillment. Cryptocurrency (or Centralized?) exchanges participating in the network can aggregate their liquidity and provide deeper order books and new trading pairs to their users."

At the end I'd just say "`xud` encompasses the following components".